src/ensembl/src/content/app/entity-viewer/shared/rest/rest-data-fetchers/transcriptData.ts

```diff
@@ -1,114 +1,58 @@
-import { restGeneAdaptor } from 'src/content/app/entity-viewer/shared/rest/rest-adaptors/rest-gene-adaptor';
 import { restTranscriptAdaptor } from 'src/content/app/entity-viewer/shared/rest/rest-adaptors/rest-transcript-adaptor';
 
-import { Gene } from 'src/content/app/entity-viewer/types/gene';
 import { Transcript } from 'src/content/app/entity-viewer/types/transcript';
 
-// the in-response types below describe shape of features retrieved from the rest /overlap endpoint
-export type GeneInResponse = {
-  feature_type: 'gene';
-  id: string;
-  biotype: string;
-  external_name: string;
-  seq_region_name: string;
-  strand: 1 | -1;
-  start: number;
-  end: number;
-};
-
 export type TranscriptInResponse = {
-  feature_type: 'transcript';
+  object_type: 'Transcript';
   id: string;
-  external_name: string;
   biotype: string;
   seq_region_name: string;
   strand: 1 | -1;
   Parent: string; // gene id
   start: number;
   end: number;
-};
-
-export type ExonInResponse = {
-  feature_type: 'exon';
-  id: string;
-  Parent: string; // transcript id
-  start: number;
-  end: number;
-};
-
-type CDSInResponse = {
-  feature_type: 'cds';
-  id: string;
-  Parent: string; // transcript id
-  start: number;
-  end: number;
+  is_canonical: number;
+  Translation?: TranslationInResponse;
+  logic_name: string;
+  source: string;
+  display_name: string;
+  species: string;
+  db_type: string;
```

Comment:
Why do you list fields that we are not using and thus do not care about (`db_type`, `logic_name`, `species`)? If we are not interested in them, we can safely pretend they do not exits.

Reply:
The idea was that it should be a one thing and then to be forgotten, as this is temporary. So this will avoid unnecessarily revisiting these types, to add a property that we would want to use.